collections/_posts/2022-09-19-typelevel-native.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: Typelevel Native
+category: technical
+
+meta:
+  nav: blog
+  author: armanbilge
+---
+
+We recently published several major Typelevel projects for the [Scala Native] platform, most notably [Cats Effect], [FS2], and [http4s]. This blog post explores what this new platform means for the Typelevel ecosystem as well as how it works under-the-hood.
+
+### What is Scala Native?
+
+[Scala Native] is an optimizing ahead-of-time compiler for the Scala language. Put simply: it enables you to **compile Scala code directly to native executables**.
+
+It is an ambitious project following in the steps of [Scala.js]. Instead of targeting JavaScript, the Scala Native compiler targets the [LLVM] IR and uses its toolchain to generate native executables for a range of architectures, including x86, ARM, and in the near future web assembly.
```

Comment:
> in the near future web assembly.

Is there an up to date issue tracking this progress we could perhaps link to? Everything I could find seems to be about 4 years old.